glide.yaml

```diff
@@ -36,6 +36,8 @@ import:
   version: a389bdde4dd695d414e47b755e95e72b7826432c
 - package: github.com/go-ini/ini
   version: c787282c39ac1fc618827141a1f762240def08a3
+- package: github.com/ashcrow/osrelease
+  version: 9b292693c55c791c79825cdce5b7e0b3b6da6e88
```

Comment:
@ashcrow do you mind tagging a `v1` for us?

Reply:
Will do!

Reply:
https://github.com/ashcrow/osrelease/releases/tag/v1

Reply:
fixed